### PR TITLE
Added code instructions and a description of .gitignore, for completeness

### PR DESCRIPTION
**Pull request by thomasbd14**
Thank you @Sebastian-Santana-Ort for flagging that this was needed.
If Sebastian or anyone else is free to review, please feel do so.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ This project is to fullfill the first milestone of the collaborate project for t
 -   **TEAMWORK.md**: The group teamwork contract, outlining our agreed terms for the project
 -   **troubleshooting-1.Rmd**: A completed class-assigned exercise, which focused on correcting errors in markdown
 -   **troubleshooting-1.md**: The output file of the troubleshooting Rdocument
+-   **.gitignore** is used by git to avoid tracking superfluous files.
+
+## Running the code
+-   **troubleshooting-1.rmd** can be run by opening the file in Rstudio, then clicking "knit". This will re-generate **troubleshooting-1.md** as it appears in this directory.
```
Comment (wangyb-A) [inline README.md]: I think tidyverse needs to be installed if the user wants to run the code locally, with command `install.packages(“tidyverse”)` in R session @thomasbd14

**Review by wangyb-A — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (thomasbd14) [reply README.md]: Thank you for the reminder! I have added this in https://github.com/stat545ubc-2023/collaborative-group24/pull/19/commits/949ba3703d955086c3c46a891be726a3e1e79909

**Review by wangyb-A — APPROVED** [review]
All good!

**Comment by johnnywang3739** [comment]
Nice job guys!

**Pull request merged**